jax/_src/numpy/lax_numpy.py

```diff
@@ -2342,6 +2342,23 @@ def _pad_edge(array, pad_width):
   return array
 
 
+def _pad_stats(array, pad_width, stat_func):
```

Comment:
Have you thought about handling the optional `stat_length` argument? If you'd prefer, feel free to leave it as a TODO.